tests/by-util/test_chroot.rs

```diff
@@ -67,6 +67,11 @@ fn test_preference_of_userspec() {
     println!("result.stdout {}", result.stdout);
     println!("result.stderr = {}", result.stderr);
     let username = result.stdout.trim_end();
+
+    let result = scene.cmd("id -g -n").run();
```

Comment:
CI fails on this
```
run: id -g -n
thread 'test_chroot::test_preference_of_userspec' panicked at 'called `Result::unwrap()` on an `Err` value: Os { code: 2, kind: NotFound, message: "No such file or directory" }', tests/common/util.rs:562:14
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

Reply:
You can do it this way:
https://github.com/uutils/coreutils/blob/master/tests/by-util/test_whoami.rs#L41-L42

Reply:
Awesome, thanks!